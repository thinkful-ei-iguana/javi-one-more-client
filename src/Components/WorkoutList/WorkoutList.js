import React from 'react'
import WorkoutItem from '../WorkoutItem/WorkoutItem.js'
import '../WorkoutPage/workoutPage.css'

export default function WorkoutList(props){

    
        return (
            <div className="your-workouts-container">
                <h2 className="your-workouts-title">Your Workouts</h2>
                {props.workouts.map(workout => 
                    <WorkoutItem history={props.history}
                        key={workout.id}
                        {...workout}
                    />
                    )}
                    
            </div>
        )
    }


