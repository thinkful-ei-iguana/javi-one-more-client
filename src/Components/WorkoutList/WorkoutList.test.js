import React from 'react';
import ReactDOM from 'react-dom';
import WorkoutList from '../WorkoutList/WorkoutList';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<WorkoutList />, div);
  ReactDOM.unmountComponentAtNode(div);
});