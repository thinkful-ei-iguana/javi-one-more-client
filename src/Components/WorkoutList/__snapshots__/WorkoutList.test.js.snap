// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /workouts renders the UI as expected 1`] = `
<div
  className="your-workouts-container"
>
  <h2
    className="your-workouts-title"
  >
    Your Workouts
  </h2>
</div>
`;
